features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Listing/CommonListing.jsx

```diff
@@ -97,38 +97,14 @@ class CommonListing extends React.Component {
         this.setState({ listType: value });
     };
 
-    /**
-     * Returns the appropriate rendering depending on the type of view being used
-     *
-     * @param {string} type Listing type
-     * @param {object} apis APIs object
-     * @param {object} classes Classes prop
-     * @returns {Component}
-     * @memberof CommonListing
-     */
-    view = (type, apis, classes) => {
-        const { path } = this.props;
-        return type === 'grid' ? (
-            <Grid container className={classes.ListingWrapper}>
-                {apis.list.map(api => (
-                    <ApiThumb api={api} key={api.id} path={path} />
-                ))}
-            </Grid>
-        ) : (
-            <React.Fragment>
-                <ApiTableView apis={apis.list} />
-            </React.Fragment>
-        );
-    }
-
     /**
      *
      * @inheritdoc
      * @returns {React.Component} @inheritdoc
      * @memberof CommonListing
      */
     render() {
-        const { apis } = this.props;
+        const { apis, kind } = this.props;
```

Comment:
shall we combine this with the de-structuring in line 109

Reply:
Fixed